Aseprite.js

```diff
@@ -224,6 +227,34 @@ class Aseprite {
     this.colorDepth === 8 ? palette.index = this.paletteIndex : '';
     return palette;
   }
+  readSliceChunk() {
+    const numSliceKeys = this.readNextDWord();
+    const flags = this.readNextDWord();
+    const patch = (flags & 1) !== 0 ? {} : null
+    const pivot = (flags & 2) !== 0 ? {} : null
+    this.skipBytes(4);
+    const name = this.readNextString()
+    const keys = []
+    for(let i = 0; i < numSliceKeys; i ++) {
+      const frameNumber = this.readNextDWord();
```

Comment:
Could remedy the patch and pivot issue by maybe adding an object in this loop to be pushed onto the slices property so you can decide if it needs added or not.

Reply:
Got it, fixed with a59c8a6